dev/ManagedConnectors.md

```diff
@@ -144,6 +144,89 @@ ocm login --token <offline_token>
 ./cos-tools/bin/create-cluster-secret <cluster_id>
 ```
 
+### Step 6: Update Job
+
+There's no automatic CI/CD mechanism in Managed Connectors to update the operators with the newest version.
+
+The only way at the moment is to configure a Kubernetes [CronJob](https://v1-21.docs.kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/)
+that kills the existing operator pods, so that the deployment triggers the pull of the newest image (if existing) before recreating it.
```

Comment:
"reschedule" instead of "kill"

Reply:
Updated (I used "delete" which is the exact wording)